Dockerfile

```diff
@@ -2,110 +2,92 @@
 # STAGE 1.1: builder frontend
 ###################
 
-FROM node:12.20.1-alpine as frontend
+FROM metabase/ci:java-11-lein-2.9.6-clj-1.10.3.822-04-22-2021 as frontend
```

Comment:
Just to confirm, 39 builds fine with JDK 11?

Reply:
yup :)